docs/index.md

```diff
@@ -44,7 +44,7 @@ After that you can start executing queries:
 
 ```go
 // Check if connection credentials are valid and PostgreSQL is up and running.
-if err := db.Ping(); err != nil {
+if err := db.Ping(db.Context()); err != nil {
```

Comment:
Please replace `db.Context()` -> `ctx` since that is what you are supposed to do. `db.Context` is likely to go away some day.